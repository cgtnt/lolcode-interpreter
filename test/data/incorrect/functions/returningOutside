HAI

FOUND YR 5

KTHXBYE
