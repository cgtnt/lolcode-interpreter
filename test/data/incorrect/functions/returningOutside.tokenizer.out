@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_RETURN_VAL:FOUND YR-@3:T_T_INT:5-@5:T_COMMAND_TERMINATOR:
-@5:T_END:KTHXBYE-@6:T_COMMAND_TERMINATOR:
-@6:T_EOF:
